llvm/include/llvm/ADT/APInt.h

```diff
@@ -2263,6 +2263,10 @@ APInt mulhs(const APInt &C1, const APInt &C2);
 /// Returns the high N bits of the multiplication result.
 APInt mulhu(const APInt &C1, const APInt &C2);
 
+/// Compute X^N for N>=0.
+/// 0^0 would return 1.
```

Comment:
ubernit: maybe something more descriptive like:
```suggestion
/// 0^0 is supported and returns 1.
```